core/src/test/java/com/linecorp/armeria/client/cookie/DefaultCookieJarTest.java

```diff
@@ -189,15 +189,15 @@ void cookieState() {
         final Cookie cookie = Cookie.of("name", "value");
         final Cookie expectCookie = Cookie.builder("name", "value").domain("foo.com").path("/").build();
 
-        assertThat(cookieJar.state(cookie)).isEqualTo(CookieJar.CookieState.NON_EXISTENT);
+        assertThat(cookieJar.state(cookie)).isEqualTo(CookieState.NON_EXISTENT);
 
         cookieJar.set(foo, Cookies.of(cookie));
-        assertThat(cookieJar.state(expectCookie)).isEqualTo(CookieJar.CookieState.EXISTENT);
+        assertThat(cookieJar.state(expectCookie)).isEqualTo(CookieState.EXISTENT);
 
         final Cookie expireCookie = expectCookie.toBuilder().maxAge(1).build();
         cookieJar.set(foo, Cookies.of(cookie.toBuilder().maxAge(1).build()));
-        assertThat(cookieJar.state(expireCookie)).isEqualTo(CookieJar.CookieState.EXISTENT);
+        assertThat(cookieJar.state(expireCookie)).isEqualTo(CookieState.EXISTENT);
         assertThat(cookieJar.state(expireCookie, System.currentTimeMillis() + 1000))
-                .isEqualTo(CookieJar.CookieState.EXPIRED);
+                .isEqualTo(CookieState.EXPIRED);
```

Comment:
How about adding an assertion for the off-by-one case I mentioned above? e.g.

- Must be considered unexpired if timePassedMillis == maxAgeMillis == 1000
- Must be considered expired if timePassedMillis == 1001 and maxAgeMillis = 1000